.github/workflows/ci.yml

```diff
@@ -101,6 +101,13 @@ jobs:
 
   check-clippy:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust-version: [
+          stable,
+          beta
```

Comment:
If we start running `clippy` on `beta` doesn't it make it redundant to run on `stable`? I.e what could be the use case that `stable` finds lints that `beta` doesn't?

Reply:
The `stable` job will be required to be green. The `beta` one can fail!

If we only run beta, we have the same problem where lints appear out of nowhere and fail our CI everytime beta gets cut.

Reply:
ah great! Thanks for clarifying Thomas :)